bus-api/pom.xml

```diff
@@ -45,7 +45,7 @@
     <dependency>
       <groupId>org.hawkular.bus</groupId>
       <artifactId>hawkular-bus-common</artifactId>
-      <version>0.0.5</version>
+      <version>0.3.0.Final</version>
```

Comment:
You should not be hardcoding component versions like this.  Extract out into a <version.org.hawkular.bus> property at least in the top parent pom.xml of your component (we should probably put these in parent pom in the future)